bundle/bundle.go

```diff
@@ -2,15 +2,57 @@ package bundle
 
 import (
 	"embed"
+	"fmt"
+	"os"
+	"sync"
+	"sync/atomic"
+
+	"github.com/open-policy-agent/opa/v1/bundle"
 
 	rio "github.com/styrainc/regal/internal/io"
 )
 
 // Bundle FS will include the tests as well, but since that has negligible impact on the size of the binary,
-// it's preferable to filter them out from the bundle than to e.g. create a separate directory for tests
-//
-//go:embed *
-var Bundle embed.FS
+// it's preferable to filter them out from the bundle than to e.g. create a separate directory for tests.
+var (
+	//go:embed *
+	regalBundle    embed.FS
+	devPath        = os.Getenv("REGAL_BUNDLE_PATH")
+	lastErrMsg     = atomic.Pointer[string]{}
+	EmbeddedBundle = sync.OnceValue(func() *bundle.Bundle {
+		return rio.MustLoadRegalBundleFS(regalBundle)
+	})
+)
+
+// LoadedBundle contains the Regal bundle.
+func LoadedBundle() *bundle.Bundle {
+	// For development, allow bundle to be loaded dynamically from path instead
+	// of the normal one embedded in the compiled binary. This allows editing e.g.
+	// LSP policies while the language server is running. This should be considered
+	// *very* experimental at this point.
+	if devPath != "" {
+		b, err := rio.LoadRegalBundlePath(devPath)
+		if err == nil {
+			if last := lastErrMsg.Load(); last != nil {
+				lastErrMsg.Store(nil)
+
+				fmt.Fprintln(os.Stderr, "development bundle back to a good state, no longer using embedded bundle")
+			}
+
+			return b
+		}
+
+		// Avoid flooding the console/logs with the same error message
```

Comment:
I guess you see this when there are issues parsing a WIP bundle file? Could add a note about that in the comment here.